NpingOps.cc won't compile without limits.h header as INT_MAX is not defined.

The same is true for charpool.cc and CHAR_MAX.

This was offered upstream:
https://github.com/nmap/nmap/pull/2509

--- nmap-7.94/nping/nping.h
+++ nmap-7.94/nping/nping.h
@@ -69,6 +69,7 @@
 #include <stdarg.h>
 #include <errno.h>
 #include <ctype.h>
+#include <limits.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 
--- nmap-7.94/charpool.cc
+++ nmap-7.94/charpool.cc
@@ -63,6 +63,7 @@
 #include <stddef.h>
 #undef NDEBUG
 #include <assert.h>
+#include <limits.h>
 
 #include "nbase.h"
 
